### PR TITLE
Adds query to analyze Long Running Queries

### PR DESCRIPTION
**Pull request by silent-lad**
This enables your to queries running for more than x Minutes (5 here). This lets you see what might be hogging up your CPU and network pool. Crucial in determining what might take your DB down in high traffic.

### CODE REVIEW

**Comment by ssrihari** [comment]
Yup, absolutely essential. Thanks @silent-lad 🙇

File: README.md [hunk 1]
```diff
@@ -94,6 +94,25 @@ ORDER BY calls DESC;
...
+  now() - pg_stat_activity.query_start AS query_time,
+  query,
+  state,
+  wait_event_type,
+  wait_event
+FROM pg_stat_activity
+WHERE (now() - pg_stat_activity.query_start) > interval '5 minutes';
+```
+
+This enables your to queries running for more than x Minutes (5 here). This gives you visibility as what might be hogging up your CPU and network pool.
```
Comment (shhivam) [inline README.md:113]: Might be a grammatical mistake here. 

Rewording suggestion:

`This enables you to know the queries ....`

Since this PR is already merged, maybe this change can be raised in another PR.

Comment (silent-lad) [reply README.md:113]: https://github.com/nilenso/postgresql-monitoring/pull/8

Yea my bad. Have raised this.
